std/parallelism.d

```diff
@@ -4293,6 +4293,24 @@ unittest
 
 //version = parallelismStressTest;
 
+private bool isNaN(float f)
+{
+    static import std.math;
+    return std.math.isNaN(f);
+}
+
+private bool isNaN(double f)
+{
+    static import std.math;
+    return std.math.isNaN(f);
+}
+
+private bool isNaN(real f)
+{
+    static import std.math;
+    return std.math.isNaN(f);
+}
```

Comment:
I thought so to, but isNaN was passed as a function to a function of parallelism. But this code has not been  run at least since isNaN became a template. This is just code trying to get the tests into a compileable state.